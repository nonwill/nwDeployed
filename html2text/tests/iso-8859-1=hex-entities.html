<!-- buildtemplates/TKPMAILINGEXTERN.tpl -->
            <!-- PROJECT_ROOT_DIR/modules/newsletters/templates/newsletteraegon/view.tpl -->

<!-- PROJECT_ROOT_DIR/modules/newsletters/templates/newsletteraegon/mail-view.tpl -->


<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">

	<head>
		
	
</head>

	<body>
		<span class="preheader" style="display:none;display:none !important;overflow:hidden;width:0;height:0;max-width:0;max-height:0;line-height:0;color:transparent;font-size: 1px;mso-hide: all;opacity: 0;"></span><!--[if !mso]><!--><meta http-equiv="X-UA-Compatible" content="IE=edge"/><!--<![endif]--><meta name="format-detection" content="telephone=no"/><meta name="viewport" content="width=device-width, initial-scale=1.0"><title>Coronavirus: de gevolgen voor je pensioenregeling bij Aegon Cappital DEELNEMER &lt;55</title><!-- PROJECT_ROOT_DIR/modules/newsletters/templates/newsletteraegon/style-mail.tpl -->






<!--[if !mso]><!-->
<link href="https://fonts.googleapis.com/css?family=Open+Sans:300, 400,600,700,800" rel="stylesheet">
<!--<![endif]-->



<style type="text/css">
	#outlook a {
		padding: 0;
	}

	body {
		width: 100% !important;
		min-width: 100%;
		-webkit-text-size-adjust: 100%;
		-ms-text-size-adjust: 100%;
		margin: 0;
		Margin: 0; 
		padding: 0;
		-moz-box-sizing: border-box;
		-webkit-box-sizing: border-box;
		box-sizing: border-box;
	}

	.ExternalClass {
		width: 100%;
	}

	.ExternalClass,
	.ExternalClass p,
	.ExternalClass span,
	.ExternalClass font,
	.ExternalClass td,
	.ExternalClass div {
		line-height: 100%;
	}

	img {
		outline: none;
		text-decoration: none;
		-ms-interpolation-mode: bicubic;
		width: auto;
		max-width: 100%;
		clear: both;
		display: block;
	}

	a img {
		border: none;
	}

	p {
		margin: 0px;
	}

	table {
		border-spacing: 0;
		border-collapse: collapse;
	}

	td {
		word-wrap: break-word;
		-webkit-hyphens: auto;
		-moz-hyphens: auto;
		hyphens: auto;
		border-collapse: collapse !important;
	}

	table, tr, td {
		padding: 0;
		vertical-align: top;
		text-align: left;
	}

	/*
		Classnames for columns in desktop view
		Overruled when `small-12` or `small-6` is added as classname
	*/
	th.large-4 {
		width: 192px;
	}

	th.large-6 {
		width: 288px;
	}

	th.large-8 {
		width: 384px;
	}

	th.large-12 {
		width: 600px;
	}

	@media screen and (max-width: 600px) {

		table {
			width: 100% !important;
		}

		table.body img {
			width: auto;
			height: auto;
		}

		th.small-6 {
			display: inline-block !important;
			width: 50% !important;
		}

		th.small-12 {
			display: inline-block !important;
			width: 100% !important;
		}

		.text-align-center {
			text-align: center !important;
		}
	}
</style>

		<!-- START MAILING TEMPLATE-->
		<table class="mailbody"  width="100%" align="center">
			<tr>
				<td bgcolor="" valign="top" align="left">

					<!-- PROJECT_ROOT_DIR/modules/newsletters/templates/newsletteraegon/mail-header.tpl -->

<!-- START HEADER -->
<table cellpadding='0' cellspacing='0' border='0' width="100%" class="mail-header">
	<tr>
		<td>
			<table cellpadding='0' cellspacing='0' border='0' width="600" align="center" style="max-width: 600px;">
				<tr><td  height="12" style="font-size:12px;line-height:12px" >&nbsp;</td></tr>
				<tr>
					<td style="text-align: right;">
						<table cellpadding='0' cellspacing='0' border='0' width="100%;" style="font-size: 11px; line-height: 12px; color: #989898; font-family: &#039;Open Sans&#039;, Verdana, Geneva, sans-serif;">
							<tr><td  height="6" style="font-size:6px;line-height:6px" colspan="3">&nbsp;</td></tr>
							<tr>
								<td width="24"></td>
								<td class="text-align-center" style="font-size: 11px; line-height: 12px; color: #989898; font-family: &#039;Open Sans&#039;, Verdana, Geneva, sans-serif;text-align: right;">
									<a href="https://foo" style="font-size: 11px; line-height: 12px; color: #989898; font-family: &#039;Open Sans&#039;, Verdana, Geneva, sans-serif; text-decoration: none;">Bekijk de online versie</a>
								</td>
								<td width="24"></td>
							</tr>
							<tr><td  height="6" style="font-size:6px;line-height:6px" colspan="3">&nbsp;</td></tr>
						</table>
					</td>
				</tr>
				<tr><td  height="12" style="font-size:12px;line-height:12px" >&nbsp;</td></tr>
				<tr>
					<td>
						<table cellpadding='0' cellspacing='0' border='0' width="100%" bgcolor="#FFFFFF">
							<tr>
								<td>
																			<img src="https://bar/Corona.jpg" >
																	</td>
							</tr>
						</table>
					</td>
				</tr>
			</table>
		</td>
	</tr>
</table>
<!-- END HEADER -->
					<!-- PROJECT_ROOT_DIR/modules/newsletters/templates/newsletteraegon/mail-middle.tpl -->


	<!-- START CONTENT -->
	<table cellpadding='0' cellspacing='0' border='0' width="100%" class="mail-intro">
		<tr>
			<td>
				<table cellpadding='0' cellspacing='0' border='0' width="600" align="center" style="max-width: 600px;">
					<tr>
						<td>
							<table cellpadding='0' cellspacing='0' border='0' width="100%" style="font-size: 14px; line-height: 24px; color: #333333; font-family: &#039;Open Sans&#039;, sans-serif;">
																	<!-- PROJECT_ROOT_DIR/modules/newsletters/templates/newsletterblock/newsletter.tpl -->


	<tr>
		<td	 bgcolor="#dbf0fd"
			>
			<table cellpadding='0' cellspacing='0' border='0' width="100%">
								<tr>
									<td>
						<table style="border-collapse: collapse;">
							<tr><td  height="24" style="font-size:24px;line-height:24px" colspan="3">&nbsp;</td></tr>
														<tr>
								<td colspan="3">
									<a id="section-20614" name="section-20614"></a>
									<table cellpadding='0' cellspacing='0' border='0' width="100%">
																					<!-- PROJECT_ROOT_DIR/modules/newsletters/templates/newsletteralinea/newsletter.tpl -->

<tr>
	<td>
		<table cellpadding='0' cellspacing='0' border='0' width="100%">
			<tr>
				<td width="24"></td>
				<td>
					<a id="section-20613" name="section-20613"></a>
					<table cellpadding='0' cellspacing='0' border='0' width="100%" align="center" style="font-size: 14px; line-height: 24px; color: #333333; font-family: &#039;Open Sans&#039;, sans-serif; margin:auto; text-align:left;">
																								<tr><td  height="24" style="font-size:24px;line-height:24px" >&nbsp;</td></tr>
																		<tr>
							<td style="font-size: 14px; line-height: 24px; color: #333333; font-family: &#039;Open Sans&#039;, sans-serif;">
																											<p style="font-size: 14px; line-height: 24px; color: #333333; font-family: 'Open Sans', sans-serif; margin-top: 0; margin-bottom:12px; font-weight: 600;">Beste,</p>

<p style="font-size: 14px; line-height: 24px; color: #333333; font-family: 'Open Sans', sans-serif; margin-top: 0; margin-bottom:12px; font-weight: 600;">Het coronavirus houdt ons land in de greep. We hopen dat het goed gaat met jou en je naasten. Ongetwijfeld heb je al veel mails gehad van bedrijven over de gevolgen van het virus. Toch vragen ook wij even je aandacht. In dit geval voor je pensioen en de gevolgen die het coronavirus daarop heeft.</p>

																								</td>
						</tr>
											</table>
				</td>
				<td width="24"></td>
			</tr>
			<tr><td  height="24" style="font-size:24px;line-height:24px" colspan="3">&nbsp;</td></tr>
		</table>
	</td>
</tr>

										
									</table>
								</td>
							</tr>
						</table>
					</td>

								</tr>
							</table>
		</td>
	</tr>
			<tr><td  height="24" style="font-size:24px;line-height:24px" >&nbsp;</td></tr>
									
							</table>
						</td>
					</tr>
				</table>
			</td>
		</tr>
	</table>
	<!-- END CONTENT -->

	<!-- START CONTENT -->
	<table cellpadding='0' cellspacing='0' border='0' width="100%" class="mail-middle">
		<tr>
			<td>
				<table cellpadding='0' cellspacing='0' border='0' width="600" align="center" style="max-width: 600px;">
					<tr>
						<td>
							<table cellpadding='0' cellspacing='0' border='0' width="100%" style="font-size: 14px; line-height: 24px; color: #333333; font-family: &#039;Open Sans&#039;, sans-serif;">
																																			<!-- PROJECT_ROOT_DIR/modules/newsletters/templates/newsletterblock/newsletter.tpl -->


	<tr>
		<td	 bgcolor="#dbf0fd"
			>
			<table cellpadding='0' cellspacing='0' border='0' width="100%">
								<tr>
									<td>
						<table style="border-collapse: collapse;">
							<tr><td  height="24" style="font-size:24px;line-height:24px" colspan="3">&nbsp;</td></tr>
														<tr>
								<td colspan="3">
									<a id="section-20615" name="section-20615"></a>
									<table cellpadding='0' cellspacing='0' border='0' width="100%">
																					<!-- PROJECT_ROOT_DIR/modules/newsletters/templates/newsletteralinea/newsletter.tpl -->

<tr>
	<td>
		<table cellpadding='0' cellspacing='0' border='0' width="100%">
			<tr>
				<td width="24"></td>
				<td>
					<a id="section-20616" name="section-20616"></a>
					<table cellpadding='0' cellspacing='0' border='0' width="100%" align="center" style="font-size: 14px; line-height: 24px; color: #333333; font-family: &#039;Open Sans&#039;, sans-serif; margin:auto; text-align:left;">
													<tr>
								<td style="font-size: 24px; line-height: 28px; color: #3395D3; font-family: &#039;Open Sans&#039;, sans-serif color:#00304C;">Beleggingen en het coronavirus</td>
							</tr>
																								<tr><td  height="24" style="font-size:24px;line-height:24px" >&nbsp;</td></tr>
																		<tr>
							<td style="font-size: 14px; line-height: 24px; color: #333333; font-family: &#039;Open Sans&#039;, sans-serif;">
																											<p style="font-size: 14px; line-height: 24px; color: #333333; font-family: 'Open Sans', sans-serif; margin-top: 0; margin-bottom:12px;">Jouw&nbsp;pensioenpremie wordt belegd. De afgelopen periode zijn de beurskoersen flink gedaald door de uitbraak van het coronavirus. Dus ook jouw beleggingen zijn minder waard geworden. We kunnen ons goed voorstellen dat je daarvan geschrokken bent. Dit soort waardedalingen komen echter vaker voor. Denk aan de kredietcrisis van 2008. En de internetbubbel van begin deze eeuw. De ervaring leert ook dat markten zich herstellen. Het geld dat je belegt, is bestemd voor je pensioen. Het is daarom belangrijk de lange termijn voor ogen te houden. Goed om te weten: wij houden hier voor jouw pensioen ook al rekening mee. Onze manier van beleggen zorgt ervoor dat wij tussentijdse bewegingen -zoals die nu door het coronavirus veroorzaakt worden- op kunnen vangen.</p>

																								</td>
						</tr>
											</table>
				</td>
				<td width="24"></td>
			</tr>
			<tr><td  height="24" style="font-size:24px;line-height:24px" colspan="3">&nbsp;</td></tr>
		</table>
	</td>
</tr>

																					<!-- PROJECT_ROOT_DIR/modules/newsletters/templates/newsletterbuttons/newsletter.tpl -->

	<tr>
		<td>
			<table cellpadding='0' cellspacing='0' border='0' width="100%">
				<tr>
					<td width="24"></td>
					<td>
						<a id="section-20639" name="section-20639"></a>
						<table cellpadding='0' cellspacing='0' border='0' width="100%">
							<tr>
								<td>
																			<!-- PROJECT_ROOT_DIR/modules/newsletters/templates/nbcalltoaction/newsletter-small.tpl -->


<table cellpadding='0' cellspacing='0' border='0' style="display: inline-block !important;">
	<tr>
		<td bgcolor="#3395D3" style="text-align:center; width: 100%;">
			<a style="font-size: 14px; line-height: 24px; color: #333333; font-family: &#039;Open Sans&#039;, sans-serif; line-height:1em; font-size:16px; color:#FFFFFF;text-decoration:none; background: #3395D3; display:block; border-style: solid; border-color:#3395D3; border-width: 6px 12px; margin: 0;"
			   href="https://baz" target="_blank">
				Meer weten? Lees de update van het Aegon Investment Office.
			</a>
		</td>
	</tr>
</table>
&nbsp;									
								</td>
							</tr>
						</table>
					</td>
					<td width="24"></td>
				</tr>
			</table>
		</td>
	</tr>
	<tr><td  height="24" style="font-size:24px;line-height:24px" >&nbsp;</td></tr>
										
									</table>
								</td>
							</tr>
						</table>
					</td>

								</tr>
							</table>
		</td>
	</tr>
			<tr><td  height="24" style="font-size:24px;line-height:24px" >&nbsp;</td></tr>
																																				<!-- PROJECT_ROOT_DIR/modules/newsletters/templates/newsletterblock/newsletter.tpl -->


	<tr>
		<td	 bgcolor="#dbf0fd"
			>
			<table cellpadding='0' cellspacing='0' border='0' width="100%">
								<tr>
									<td>
						<table style="border-collapse: collapse;">
							<tr><td  height="24" style="font-size:24px;line-height:24px" colspan="3">&nbsp;</td></tr>
														<tr>
								<td colspan="3">
									<a id="section-20618" name="section-20618"></a>
									<table cellpadding='0' cellspacing='0' border='0' width="100%">
																					<!-- PROJECT_ROOT_DIR/modules/newsletters/templates/newsletteralinea/newsletter.tpl -->

<tr>
	<td>
		<table cellpadding='0' cellspacing='0' border='0' width="100%">
			<tr>
				<td width="24"></td>
				<td>
					<a id="section-20620" name="section-20620"></a>
					<table cellpadding='0' cellspacing='0' border='0' width="100%" align="center" style="font-size: 14px; line-height: 24px; color: #333333; font-family: &#039;Open Sans&#039;, sans-serif; margin:auto; text-align:left;">
													<tr>
								<td style="font-size: 24px; line-height: 28px; color: #3395D3; font-family: &#039;Open Sans&#039;, sans-serif color:#00304C;">Geen beperking in de dekking door coronavirus</td>
							</tr>
																								<tr><td  height="24" style="font-size:24px;line-height:24px" >&nbsp;</td></tr>
																		<tr>
							<td style="font-size: 14px; line-height: 24px; color: #333333; font-family: &#039;Open Sans&#039;, sans-serif;">
																											<p style="font-size: 14px; line-height: 24px; color: #333333; font-family: 'Open Sans', sans-serif; margin-top: 0; margin-bottom:12px;">Een aantal media heeft bericht dat het nabestaandenpensioen in gevaar zou zijn door het coronavirus. Wij kunnen je gerust stellen: het coronavirus heeft geen gevolgen voor de dekking van de risico&rsquo;s van overlijden en arbeidsongeschiktheid bij Aegon Cappital. Wat er voor jou precies geregeld is, zie je op <a style="font-size: 14px; line-height: 24px; color: #333333; font-family: 'Open Sans', sans-serif; color:#3395D3;" href="https://blah" target="_blank" title="Mijn Aegon Cappital">Mijn Aegon Cappital</a>.</p>

																								</td>
						</tr>
											</table>
				</td>
				<td width="24"></td>
			</tr>
			<tr><td  height="24" style="font-size:24px;line-height:24px" colspan="3">&nbsp;</td></tr>
		</table>
	</td>
</tr>

										
									</table>
								</td>
							</tr>
						</table>
					</td>

								</tr>
							</table>
		</td>
	</tr>
			<tr><td  height="24" style="font-size:24px;line-height:24px" >&nbsp;</td></tr>
																																				<!-- PROJECT_ROOT_DIR/modules/newsletters/templates/newsletterblock/newsletter.tpl -->


	<tr>
		<td	 bgcolor="#dbf0fd"
			>
			<table cellpadding='0' cellspacing='0' border='0' width="100%">
								<tr>
									<td>
						<table style="border-collapse: collapse;">
							<tr><td  height="24" style="font-size:24px;line-height:24px" colspan="3">&nbsp;</td></tr>
														<tr>
								<td colspan="3">
									<a id="section-20619" name="section-20619"></a>
									<table cellpadding='0' cellspacing='0' border='0' width="100%">
																					<!-- PROJECT_ROOT_DIR/modules/newsletters/templates/newsletteralinea/newsletter.tpl -->

<tr>
	<td>
		<table cellpadding='0' cellspacing='0' border='0' width="100%">
			<tr>
				<td width="24"></td>
				<td>
					<a id="section-20621" name="section-20621"></a>
					<table cellpadding='0' cellspacing='0' border='0' width="100%" align="center" style="font-size: 14px; line-height: 24px; color: #333333; font-family: &#039;Open Sans&#039;, sans-serif; margin:auto; text-align:left;">
													<tr>
								<td style="font-size: 24px; line-height: 28px; color: #3395D3; font-family: &#039;Open Sans&#039;, sans-serif color:#00304C;">Arbeidsduurverkorting en je pensioen </td>
							</tr>
																								<tr><td  height="24" style="font-size:24px;line-height:24px" >&nbsp;</td></tr>
																		<tr>
							<td style="font-size: 14px; line-height: 24px; color: #333333; font-family: &#039;Open Sans&#039;, sans-serif;">
																											<p style="font-size: 14px; line-height: 24px; color: #333333; font-family: 'Open Sans', sans-serif; margin-top: 0; margin-bottom:12px;">Heeft je werkgever arbeidsduurverkorting voor je aangevraagd? Dan kan dit gevolgen hebben voor je pensioenopbouw. Je bouwt&nbsp;alleen pensioen op over de uren die je contractueel werkt. Ook is je nabestaandenpensioen in dit geval tegen een lager bedrag verzekerd. De overheid kijkt naar maatregelen om deze nadelige gevolgen voor bedrijven en werknemers zoveel mogelijk te beperken. Als je hier vragen over hebt, kan je deze het beste stellen aan je werkgever.</p>

																								</td>
						</tr>
											</table>
				</td>
				<td width="24"></td>
			</tr>
			<tr><td  height="24" style="font-size:24px;line-height:24px" colspan="3">&nbsp;</td></tr>
		</table>
	</td>
</tr>

										
									</table>
								</td>
							</tr>
						</table>
					</td>

								</tr>
							</table>
		</td>
	</tr>
			<tr><td  height="24" style="font-size:24px;line-height:24px" >&nbsp;</td></tr>
																																																							<!-- PROJECT_ROOT_DIR/modules/newsletters/templates/newsletterblock/newsletter.tpl -->


	<tr>
		<td	 bgcolor="#dbf0fd"
			>
			<table cellpadding='0' cellspacing='0' border='0' width="100%">
								<tr>
									<td>
						<table style="border-collapse: collapse;">
							<tr><td  height="24" style="font-size:24px;line-height:24px" colspan="3">&nbsp;</td></tr>
														<tr>
								<td colspan="3">
									<a id="section-20624" name="section-20624"></a>
									<table cellpadding='0' cellspacing='0' border='0' width="100%">
																					<!-- PROJECT_ROOT_DIR/modules/newsletters/templates/newsletteralinea/newsletter.tpl -->

<tr>
	<td>
		<table cellpadding='0' cellspacing='0' border='0' width="100%">
			<tr>
				<td width="24"></td>
				<td>
					<a id="section-20625" name="section-20625"></a>
					<table cellpadding='0' cellspacing='0' border='0' width="100%" align="center" style="font-size: 14px; line-height: 24px; color: #333333; font-family: &#039;Open Sans&#039;, sans-serif; margin:auto; text-align:left;">
													<tr>
								<td style="font-size: 24px; line-height: 28px; color: #3395D3; font-family: &#039;Open Sans&#039;, sans-serif color:#00304C;">We zijn nog steeds bereikbaar</td>
							</tr>
																								<tr><td  height="24" style="font-size:24px;line-height:24px" >&nbsp;</td></tr>
																		<tr>
							<td style="font-size: 14px; line-height: 24px; color: #333333; font-family: &#039;Open Sans&#039;, sans-serif;">
																											<p style="font-size: 14px; line-height: 24px; color: #333333; font-family: 'Open Sans', sans-serif; margin-top: 0; margin-bottom:12px;">We kunnen ons goed voorstellen dat je je zorgen maakt. Ook in deze moeilijke tijden staan we voor je klaar. Het kan iets langer duren voordat je ons aan de lijn hebt. Bel ons gewoon als je een dringende vraag hebt. Of stel je vraag via de chat. Wel zo makkelijk!&nbsp;Is je vraag minder dringend? Stuur dan een e-mail. Je vindt onze contactgegevens op <a style="font-size: 14px; line-height: 24px; color: #333333; font-family: 'Open Sans', sans-serif; color:#3395D3;" href="https://frob" target="_blank">www.aegoncappital.nl/contact</a>.</p>

<p style="font-size: 14px; line-height: 24px; color: #333333; font-family: 'Open Sans', sans-serif; margin-top: 0; margin-bottom:12px;"><strong style="font-size: 14px; line-height: 24px; color: #333333; font-family: 'Open Sans', sans-serif;">Met vriendelijke groet,</strong></p>

<p style="font-size: 14px; line-height: 24px; color: #333333; font-family: 'Open Sans', sans-serif; margin-top: 0; margin-bottom:12px;"><strong style="font-size: 14px; line-height: 24px; color: #333333; font-family: 'Open Sans', sans-serif;">Frits Bart</strong><br />
Directeur Aegon Cappital</p>

																								</td>
						</tr>
											</table>
				</td>
				<td width="24"></td>
			</tr>
			<tr><td  height="24" style="font-size:24px;line-height:24px" colspan="3">&nbsp;</td></tr>
		</table>
	</td>
</tr>

																					<!-- PROJECT_ROOT_DIR/modules/newsletters/templates/newsletterbuttons/newsletter.tpl -->

	<tr>
		<td>
			<table cellpadding='0' cellspacing='0' border='0' width="100%">
				<tr>
					<td width="24"></td>
					<td>
						<a id="section-20637" name="section-20637"></a>
						<table cellpadding='0' cellspacing='0' border='0' width="100%">
							<tr>
								<td>
																			<!-- PROJECT_ROOT_DIR/modules/newsletters/templates/nbcalltoaction/newsletter-small.tpl -->


<table cellpadding='0' cellspacing='0' border='0' style="display: inline-block !important;">
	<tr>
		<td bgcolor="#3395D3" style="text-align:center; width: 100%;">
			<a style="font-size: 14px; line-height: 24px; color: #333333; font-family: &#039;Open Sans&#039;, sans-serif; line-height:1em; font-size:16px; color:#FFFFFF;text-decoration:none; background: #3395D3; display:block; border-style: solid; border-color:#3395D3; border-width: 6px 12px; margin: 0;"
			   href="https://nob" target="_blank">
				English version
			</a>
		</td>
	</tr>
</table>
&nbsp;									
								</td>
							</tr>
						</table>
					</td>
					<td width="24"></td>
				</tr>
			</table>
		</td>
	</tr>
	<tr><td  height="24" style="font-size:24px;line-height:24px" >&nbsp;</td></tr>
										
									</table>
								</td>
							</tr>
						</table>
					</td>

								</tr>
							</table>
		</td>
	</tr>
									
							</table>
						</td>
					</tr>
				</table>
			</td>
		</tr>
	</table>
	<!-- END CONTENT -->

					<!-- PROJECT_ROOT_DIR/modules/newsletters/templates/newsletteraegon/mail-footer.tpl -->


<!-- START FOOTER -->
<table cellpadding='0' cellspacing='0' border='0' width="100%" class="mail-footer">
	<tr>
		<td>
			<table cellpadding='0' cellspacing='0' border='0' width="600" align="center" style="max-width: 600px;">
				<tr>
					<td width="24"></td>
					<td>
						<table cellpadding='0' cellspacing='0' border='0' width="100%" style="font-size: 11px; line-height: 12px; color: #989898; font-family: &#039;Open Sans&#039;, Verdana, Geneva, sans-serif; margin:auto;">
							<tr><td  height="12" style="font-size:12px;line-height:12px" >&nbsp;</td></tr>
							<tr>
								<td class="text-align-center" style="text-align:left font-size: 11px; line-height: 12px; color: #989898; font-family: &#039;Open Sans&#039;, Verdana, Geneva, sans-serif;">
									<a title="&copy; 2020" href="https://copy" style="font-size: 11px; line-height: 12px; color: #989898; font-family: &#039;Open Sans&#039;, Verdana, Geneva, sans-serif; text-decoration: none;">&copy; 2020 Aegon Cappital</a>
									&nbsp; &#8226; &nbsp;
									<a title="Disclaimer" href="https://disc" style="font-size: 11px; line-height: 12px; color: #989898; font-family: &#039;Open Sans&#039;, Verdana, Geneva, sans-serif; text-decoration: none;">Disclaimer</a>
									&nbsp; &#8226; &nbsp;
									<a title="Privacy" href="https://priv" style="font-size: 11px; line-height: 12px; color: #989898; font-family: &#039;Open Sans&#039;, Verdana, Geneva, sans-serif; text-decoration: none;">Privacy</a>
									&nbsp; &#8226; &nbsp;
									<a title="Afmelden" href="https://optout" style="font-size: 11px; line-height: 12px; color: #989898; font-family: &#039;Open Sans&#039;, Verdana, Geneva, sans-serif; text-decoration: none;">Afmelden</a>
																	</td>
							</tr>
							<tr><td  height="12" style="font-size:12px;line-height:12px" >&nbsp;</td></tr>
						</table>
					</td>
					<td width="24"></td>
				</tr>
								<tr><td  height="12" style="font-size:12px;line-height:12px" >&nbsp;</td></tr>
			</table>
		</td>
	</tr>
</table>
<!-- END FOOTER -->

				</td>
			</tr>
		</table>
		<!-- END MAILING TEMPLATE -->

		<!-- PREVENT GMAIL ON IOS FONT SIZE MANIPULATION -->
		<div style="display:none; white-space:nowrap; font:15px courier; line-height:0;"> &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;
			&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;
			&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;
		</div>
		<img src="https://gif" alt="" style="display:none;display:none !important;overflow:hidden;width:0;height:0;max-width:0;max-height:0;line-height:0; color:transparent; font-size: 0; mso-hide: all;"/>
	<!-- WEBAPP_ROOT_DIR/modules/analyticsbase/templates/module/tracking.tpl -->


</body>

</html>

    
