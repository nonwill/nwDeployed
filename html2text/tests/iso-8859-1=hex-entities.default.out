




                
                       
                       _B_e_k_i_j_k_ _d_e_ _o_n_l_i_n_e
                                 _v_e_r_s_i_e
                       
                
               [https://bar/Corona.jpg]

             
              
             Beste,
             Het
             coronavirus
             houdt ons
             land in de
             greep. We
             hopen dat
             het goed
             gaat met jou
             en je
             naasten.
             Ongetwijfeld
             heb je al
             veel mails
             gehad van
             bedrijven
             over de
             gevolgen van
             het virus.
             Toch vragen
             ook wij even
             je aandacht.
             In dit geval
             voor je
             pensioen en
             de gevolgen
             die het
             coronavirus
             daarop
             heeft.
             
             

           
           Beleggingen en het
           coronavirus
            
           Jouw pensioenpremie
           wordt belegd. De
           afgelopen periode
           zijn de
           beurskoersen flink
           gedaald door de
           uitbraak van het
           coronavirus. Dus
           ook jouw
           beleggingen zijn
           minder waard
           geworden. We kunnen
           ons goed
           voorstellen dat je
           daarvan geschrokken
           bent. Dit soort
           waardedalingen
           komen echter vaker
           voor. Denk aan de
           kredietcrisis van
           2008. En de
           internetbubbel van
           begin deze eeuw. De
           ervaring leert ook
           dat markten zich
           herstellen. Het
           geld dat je belegt,
           is bestemd voor je
           pensioen. Het is
           daarom belangrijk
           de lange termijn
           voor ogen te
           houden. Goed om te
           weten: wij houden
           hier voor jouw
           pensioen ook al
           rekening mee. Onze
           manier van beleggen
           zorgt ervoor dat
           wij tussentijdse
           bewegingen -zoals
           die nu door het
           coronavirus
           veroorzaakt worden-
           op kunnen vangen.
           
           _M_e_e_r_ _w_e_t_e_n_?_ _L_e_e_s
           _d_e_ _u_p_d_a_t_e_ _v_a_n_ _h_e_t
           _A_e_g_o_n_ _I_n_v_e_s_t_m_e_n_t
                _O_f_f_i_c_e_.
            
           
           
           
           Geen beperking in de
           dekking door
           coronavirus
            
           Een aantal media
           heeft bericht dat het
           nabestaandenpensioen
           in gevaar zou zijn
           door het coronavirus.
           Wij kunnen je gerust
           stellen: het
           coronavirus heeft
           geen gevolgen voor de
           dekking van de
           risico’s van
           overlijden en
           arbeidsongeschiktheid
           bij Aegon Cappital.
           Wat er voor jou
           precies geregeld is,
           zie je op _M_i_j_n_ _A_e_g_o_n
           _C_a_p_p_i_t_a_l.
           
           
           
           Arbeidsduurverkorting
           en je pensioen
            
           Heeft je werkgever
           arbeidsduurverkorting
           voor je aangevraagd?
           Dan kan dit gevolgen
           hebben voor je
           pensioenopbouw. Je
           bouwt alleen pensioen
           op over de uren die
           je contractueel
           werkt. Ook is je
           nabestaandenpensioen
           in dit geval tegen
           een lager bedrag
           verzekerd. De
           overheid kijkt naar
           maatregelen om deze
           nadelige gevolgen
           voor bedrijven en
           werknemers zoveel
           mogelijk te beperken.
           Als je hier vragen
           over hebt, kan je
           deze het beste
           stellen aan je
           werkgever.
           
           
           
           We zijn nog steeds
           bereikbaar
            
           We kunnen ons goed
           voorstellen dat je je
           zorgen maakt. Ook in
           deze moeilijke tijden
           staan we voor je
           klaar. Het kan iets
           langer duren voordat
           je ons aan de lijn
           hebt. Bel ons gewoon
           als je een dringende
           vraag hebt. Of stel
           je vraag via de chat.
           Wel zo makkelijk! Is
           je vraag minder
           dringend? Stuur dan
           een e-mail. Je vindt
           onze contactgegevens
           op
           _w_w_w_._a_e_g_o_n_c_a_p_p_i_t_a_l_._n_l_/
           _c_o_n_t_a_c_t.
           MMeett vvrriieennddeelliijjkkee
           ggrrooeett,,
           FFrriittss BBaarrtt
           Directeur Aegon
           Cappital
           
           _E_n_g_l_i_s_h_ _v_e_r_s_i_o_n
            
           

           
          _©_ _2_0_2_0_ _A_e_g_o_n_ _C_a_p_p_i_t_a_l   •   _D_i_s_c_l_a_i_m_e_r   •   _P_r_i_v_a_c_y   •  
          _A_f_m_e_l_d_e_n
           
         

                                                           

